js/src/constants.js

```diff
@@ -79,3 +79,25 @@ export const CATEGORY_CONDITION_SELECT_TYPES = {
 };
 
 export const CATEGORIES_TO_SHOW_IN_TOOLTIP = 5;
+
+// Assets of Performance Max Campaign related
+export const ASSET_KEY = {
```

Comment:
Since this constants are a set of keys I wonder if we should rename them as ASSET_KEYS, ASSET_GROUP_KEYS, and ASSET_FORM_KEYS

Reply:
The plural is usually easier to be deduced as an array type, while the singular is usually not deducible as a specific type, so I think the singular is still more appropriate for this case. Sometimes I would use FOO_BAR_DICT (DICT for the abbr of dictionary) but it is also often too wordy.

Reply:
I understand your thoughts. But for me, it was a bit confusing `ASSET_FORM_KEY` in singular. I thought was some kind of API KEY or something like that. Was unexpected to see a set of keys inside. 

Maybe FIELDS instead of KEY would make it clearer.